src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -779,33 +779,24 @@ public Controller getController()
      * configured in the arena selector of this strategy.
      *
      * @param sstables a collection of the sstables to be assigned to arenas
-     * @param compactionFilter a filter to exclude CompactionSSTables,
+     * @param compactionFilter a bifilter (sstable, isCompacting) to exclude CompactionSSTables,
```

Comment:
Minor: I know the phrasing was already there, but it looks counter-intuitive to me as the filter should "include" the CompactionSSTables that satisfies such predicate. Obviously that means it excludes the ones that do not satisfy it, but in this case I would phrase it in terms of inclusion of what we decide to select, don't know what you think.